thefuck/rules/rails_migrations_pending.py

```diff
@@ -0,0 +1,12 @@
+import re
+
+
+SUGGESTION_REGEX = r"To resolve this issue, run:\s+(.*?)\n"
+
+
+def match(command):
+    return "Migrations are pending. To resolve this issue, run:" in command.output
```

Comment:
Nit: should this rule be used only for `bin/rspec`?

```suggestion
@for_app("bin/rspec")
def match(command):
    return "Migrations are pending. To resolve this issue, run:" in command.output
```

Reply:
No, I happen to use rspec but there are other testing libraries for Rails. Theoretically any command can trigger a `PendingMigrationError`